test/Sanitizers/tsan-libdispatch.swift

```diff
@@ -0,0 +1,43 @@
+// RUN: %target-swiftc_driver %s -g -sanitize=thread %import-libdispatch -o %t_tsan-binary
+// RUN: %target-codesign %t_tsan-binary
+// RUN: not env %env-TSAN_OPTIONS=abort_on_error=0 %target-run %t_tsan-binary 2>&1 | %FileCheck %s
+// REQUIRES: executable_test
+// REQUIRES: tsan_runtime
+// UNSUPPORTED: OS=tvos
```

Comment:
What's wrong with tvOS but not watchOS or iOS?

Reply:
I included it because the other TSan tests, e.g., tsan.swift, tsan-emptyarraystorage.swift, ..., had it.

Reply:
Hm, okay.